realsense2_camera/cfg/base_d400_params.py

```diff
@@ -5,8 +5,8 @@
 
 def add_base_params(gen, prefix):
   #             Name                                               Type    Level Description                  Default    Min     Max
-  gen.add(str(prefix) + "depth_gain",                              int_t,    1,  "Gain",                      16,        16,     248)
-  gen.add(str(prefix) + "depth_enable_auto_exposure",              bool_t,   2,  "Enable Auto Exposure",      True)
+  gen.add(str(prefix) + "depth_gain",                              int_t,    1,  "Gain",                      32,        16,     248)
+  gen.add(str(prefix) + "depth_enable_auto_exposure",              bool_t,   2,  "Enable Auto Exposure",      False)
```

Comment:
No auto exposure?

Reply:
nope, we need a constant exposure for the IR marker detection to be consistent

Reply:
FYI, FlippyTV looks a little weird now. The chicken is some blood orange color and it's pretty scary :O